Google-sandbox-best-practices-2026.md

```
@@ -0,0 +1,674 @@
+---
+
+## 目次
+
+1. [サンドボックスとは？](#1-サンドボックスとは)
+2. [全体マップ：6つのサンドボックス技術](#2-全体マップ6つのサンドボックス技術)
+3. [GKE Agent Sandbox](#3-gke-agent-sandbox--aiエージェント向け)
+4. [Gemini Code Execution](#4-gemini-code-execution--aiapi向け)
+5. [gVisor / GKE Sandbox](#5-gvisor--gke-sandbox--コンテナ向け)
+6. [Sandbox2 / SAPI](#6-sandbox2--sapi--cc向け)
+7. [V8 Sandbox](#7-v8-sandbox--ブラウザ向け)
+8. [Privacy Sandbox](#8-privacy-sandbox--廃止済み--歴史的記録)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)